tarball-ingester.tf

```
@@ -409,6 +408,16 @@ resource "aws_security_group_rule" "vpc_endpoints_from_tarball_ingester" {
   source_security_group_id = aws_security_group.tarball_ingester.id
 }
 
+resource "aws_security_group_rule" "tarball_ingester_inbound_healthcheck" {
+  description       = "Allow traffic from LB CIDR for Instance Healthcheck"
+  type              = "ingress"
+  from_port         = 9000
+  to_port           = 9000
+  protocol          = "tcp"
+  cidr_blocks       = [data.terraform_remote_state.ingest.outputs.vpc.vpc.vpc.cidr_block]
```

```suggestion
  cidr_blocks       = [data.terraform_remote_state.ingest.outputs.ingestion_subnets.cidr_block]
```
Ties it down a little bit!